adding_species_manuscript.tex

```diff
@@ -453,34 +421,65 @@ \subsection*{Additions to \stdpopsim}
-Not all of these have genetic maps or demographic models,
+two bacteria,
+%in addition to
+four primates and a common mammalian associate of primates (\textit{Canis familiaris}).
+% Elise: An aside, I find this way of describing dogs to be hilariously academic
+% Ilan: I concur :-)
+Not all of these have genetic maps or demographic models (see Figure \ref{fig:tree}),
 but this lays the framework for future contributions.
+\begin{figure}
+    \includegraphics[width=\linewidth]{figs/species_fig}
+	\caption{Phylogenetic tree of species available in the \stdpopsim catalog,
+		including the six species we published in the original release \citep[in blue]{Adrion2020}, and 15 species that have since been added (in orange).
+		Solid circles indicate species that have one (light grey) or more
+		(dark grey) demographic models and genetic maps.
+	\label{fig:tree} }
+\end{figure}
 
-A key feature added to \stdpopsim that expands the diversity of species that can
-be realistically modeled is the inclusion of recombination by gene conversion,
-which is essential for organisms such as \textit{E.\ coli}
-that lack crossing over.
-\colorbox{yellow}{[ TODO: summarize model of GC (see PR\#57) ]}
+Expanding the species catalog required adding several functionalities to the simulation framework.
+We thus upgraded the neutral simulation engine, \texttt{msprime}, from version 0.7.4 to version 1.0. \citep{Baumdicker2022}.
+This upgrade provides a number of benefits such as a discrete site model of mutation,
+so that simulated data will now %, as in real data,
+have a small proportion of sites with multiple mutations and possibly more than two alleles.
+Another key feature enabled by this upgrade was recombination by gene conversion,
+which is essential for modeling genomes of bacteria and archea.
+Gene conversion affects shorter segments than crossover recombination and creates distinct patterns of genetic diversity along the genome~\citep{Korunes2017}.
+In bacteria and archaea, homologous recombination occurs primarily through gene conversion rather than single crossover recombinations.
+As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers.
+Gene conversion in such species is implemented in \stdpopsim by setting the
+\texttt{bacterial\_recombination} flag to \texttt{True} (as a feature of the genome)
+and the  \texttt{gene\_conversion\_length} parameter to the average gene conversion tract length.
+This will result in all recombinations being simulated via gene conversion
+(no crossover recombinations), where the tract length is sampled from a geometric
+distribution whose mean is the specified length.
+For example, the model for \textit{Escherichia coli} has been updated in the \stdpopsim catalog to have a gene conversion rate of $8.9\times 10^{-11}$ with an average tract length of 345 bases \citep{Wielgoss2011,Didelot2012}.
```

Comment:
actually it has a *recombination rate* of this, but the "bacterial recombination" flag is on, which means that "recombination" is interpreted as gene conversion

Reply:
This is what I meant, but I'll use the correct terminoligy